pkgs/development/tools/build-managers/gradle/default.nix

```diff
@@ -57,28 +53,21 @@ rec {
         between the flexibility of Ant and the convenience of a
         build-by-convention behavior.
       '';
-      homepage = "http://www.gradle.org/";
+      homepage = "https://docs.gradle.org/${version}/";
```

Comment:
Is this standard? I found that most just go to the project homepage rather than docs.

Reply:
I would prefer to keep it as-is for now. Maybe we need a `docsPage` in the future. The other changes look good.

Reply:
OK, reverting.

Reply:
Fixed in 5153a6cf192